Changelog.md

```diff
@@ -6,6 +6,12 @@
 
 [0.12.0.0]: https://github.com/haskell/bytestring/compare/0.11.3.0...0.12.0.0
 
+[0.11.3.1] — March 2022
+
+* [Remove dependency on `gcc_s`](https://github.com/haskell/bytestring/pull/500)
```

Comment:
This might be slightly clearer?!

```suggestion
* [Windows: Remove problematic dependency on `gcc_s`](https://github.com/haskell/bytestring/pull/500)
```